LECT_10 (Google Storage)/dags/gcloud.py

```diff
@@ -0,0 +1,105 @@
+"""
+Upload a file to GCS
+
+Notes:
+    - all examples can be found here:
+    https://github.com/googleapis/python-storage/tree/main/samples/snippets
+"""
+from google.cloud import storage
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+from google.oauth2 import service_account
+
+import json
+import os
+import tempfile
+
+def upload_blob(bucket_name, source_file_path, destination_blob_name):
```

Comment:
I expect students to find and use LocalFilesystemToGCSOperator operator. 
This function looks like reinventing a wheel.